frontend/app/add-balance/page.tsx

```
@@ -0,0 +1,130 @@
+  const [method, setMethod] = useState<"sslcommerz" | "stripe">("sslcommerz");
+  const [loading, setLoading] = useState(false);
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    setLoading(true);
+
+    try {
+      const { data, status } = await apiRequest({
+        method: "POST",
+        url: "/add-balance",
+        data: { amount: parseFloat(amount), payment_method: method },
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)